Production is not running

Must have been a change in the last deploy?

Main error seems to be:
```
Error: Cannot find module 'tsconfig-paths/register
```

Heroku Logs:
```
2020-10-21T08:38:55.744606+00:00 heroku[worker.1]: State changed from crashed to starting
2020-10-21T08:38:59.429645+00:00 heroku[worker.1]: Starting process with command `npm start`
2020-10-21T08:39:00.111423+00:00 heroku[worker.1]: State changed from starting to up
2020-10-21T08:39:02.221035+00:00 app[worker.1]: npm info it worked if it ends with ok
2020-10-21T08:39:02.221607+00:00 app[worker.1]: npm verb cli [ '/app/.heroku/node/bin/node', '/app/.heroku/node/bin/npm', 'start' ]
2020-10-21T08:39:02.221774+00:00 app[worker.1]: npm info using npm@6.14.8
2020-10-21T08:39:02.222004+00:00 app[worker.1]: npm info using node@v14.13.1
2020-10-21T08:39:02.222177+00:00 app[worker.1]: npm verb config Skipping project config: /app/.npmrc. (matches userconfig)
2020-10-21T08:39:02.304051+00:00 app[worker.1]: npm verb run-script [ 'prestart', 'start', 'poststart' ]
2020-10-21T08:39:02.305330+00:00 app[worker.1]: npm info lifecycle hackbot@2.3.0~prestart: hackbot@2.3.0
2020-10-21T08:39:02.306033+00:00 app[worker.1]: npm info lifecycle hackbot@2.3.0~start: hackbot@2.3.0
2020-10-21T08:39:02.309985+00:00 app[worker.1]: 
2020-10-21T08:39:02.309989+00:00 app[worker.1]: > hackbot@2.3.0 start /app
2020-10-21T08:39:02.309990+00:00 app[worker.1]: > ts-node -r tsconfig-paths/register src/index.ts
2020-10-21T08:39:02.309995+00:00 app[worker.1]: 
2020-10-21T08:39:02.788111+00:00 app[worker.1]: Error: Cannot find module 'tsconfig-paths/register'
2020-10-21T08:39:02.788166+00:00 app[worker.1]: Require stack:
2020-10-21T08:39:02.788168+00:00 app[worker.1]: - internal/preload
2020-10-21T08:39:02.788169+00:00 app[worker.1]:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
2020-10-21T08:39:02.788169+00:00 app[worker.1]:     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
2020-10-21T08:39:02.788170+00:00 app[worker.1]:     at Module.require (internal/modules/cjs/loader.js:952:19)
2020-10-21T08:39:02.788170+00:00 app[worker.1]:     at Function.Module._preloadModules (internal/modules/cjs/loader.js:1217:12)
2020-10-21T08:39:02.788171+00:00 app[worker.1]:     at main (/app/node_modules/ts-node/src/bin.ts:216:20)
2020-10-21T08:39:02.788171+00:00 app[worker.1]:     at Object.<anonymous> (/app/node_modules/ts-node/src/bin.ts:512:3)
2020-10-21T08:39:02.788171+00:00 app[worker.1]:     at Module._compile (internal/modules/cjs/loader.js:1063:30)
2020-10-21T08:39:02.788172+00:00 app[worker.1]:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
2020-10-21T08:39:02.788172+00:00 app[worker.1]:     at Module.load (internal/modules/cjs/loader.js:928:32)
2020-10-21T08:39:02.788173+00:00 app[worker.1]:     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
2020-10-21T08:39:02.799520+00:00 app[worker.1]: npm verb lifecycle hackbot@2.3.0~start: unsafe-perm in lifecycle true
2020-10-21T08:39:02.799694+00:00 app[worker.1]: npm verb lifecycle hackbot@2.3.0~start: PATH: /app/.heroku/node/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/app/node_modules/.bin:/app/.heroku/node/bin:/app/.heroku/yarn/bin:/usr/local/bin:/usr/bin:/bin:/app/bin:/app/node_modules/.bin
2020-10-21T08:39:02.799834+00:00 app[worker.1]: npm verb lifecycle hackbot@2.3.0~start: CWD: /app
2020-10-21T08:39:02.799921+00:00 app[worker.1]: npm info lifecycle hackbot@2.3.0~start: Failed to exec start script
2020-10-21T08:39:02.805120+00:00 app[worker.1]: npm verb stack Error: hackbot@2.3.0 start: `ts-node -r tsconfig-paths/register src/index.ts`
2020-10-21T08:39:02.805206+00:00 app[worker.1]: npm verb stack Exit status 1
2020-10-21T08:39:02.805277+00:00 app[worker.1]: npm verb stack     at EventEmitter.<anonymous> (/app/.heroku/node/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
2020-10-21T08:39:02.805358+00:00 app[worker.1]: npm verb stack     at EventEmitter.emit (events.js:314:20)
2020-10-21T08:39:02.805424+00:00 app[worker.1]: npm verb stack     at ChildProcess.<anonymous> (/app/.heroku/node/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
2020-10-21T08:39:02.805560+00:00 app[worker.1]: npm verb stack     at ChildProcess.emit (events.js:314:20)
2020-10-21T08:39:02.805727+00:00 app[worker.1]: npm verb stack     at maybeClose (internal/child_process.js:1047:16)
2020-10-21T08:39:02.805890+00:00 app[worker.1]: npm verb stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
2020-10-21T08:39:02.806181+00:00 app[worker.1]: npm verb pkgid hackbot@2.3.0
2020-10-21T08:39:02.806417+00:00 app[worker.1]: npm verb cwd /app
2020-10-21T08:39:02.806801+00:00 app[worker.1]: npm verb Linux 4.4.0-1078-aws
2020-10-21T08:39:02.806906+00:00 app[worker.1]: npm verb argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2020-10-21T08:39:02.807013+00:00 app[worker.1]: npm verb node v14.13.1
2020-10-21T08:39:02.807108+00:00 app[worker.1]: npm verb npm  v6.14.8
2020-10-21T08:39:02.807343+00:00 app[worker.1]: npm ERR! code ELIFECYCLE
2020-10-21T08:39:02.807559+00:00 app[worker.1]: npm ERR! errno 1
2020-10-21T08:39:02.816772+00:00 app[worker.1]: npm ERR! hackbot@2.3.0 start: `ts-node -r tsconfig-paths/register src/index.ts`
2020-10-21T08:39:02.816895+00:00 app[worker.1]: npm ERR! Exit status 1
2020-10-21T08:39:02.817044+00:00 app[worker.1]: npm ERR! 
2020-10-21T08:39:02.817156+00:00 app[worker.1]: npm ERR! Failed at the hackbot@2.3.0 start script.
2020-10-21T08:39:02.817275+00:00 app[worker.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-10-21T08:39:02.817808+00:00 app[worker.1]: npm verb exit [ 1, true ]
2020-10-21T08:39:02.827024+00:00 app[worker.1]: npm timing npm Completed in 675ms
2020-10-21T08:39:02.827327+00:00 app[worker.1]: 
2020-10-21T08:39:02.827458+00:00 app[worker.1]: npm ERR! A complete log of this run can be found in:
2020-10-21T08:39:02.827550+00:00 app[worker.1]: npm ERR!     /app/.npm/_logs/2020-10-21T08_39_02_818Z-debug.log
2020-10-21T08:39:02.882612+00:00 heroku[worker.1]: Process exited with status 1
2020-10-21T08:39:02.922878+00:00 heroku[worker.1]: State changed from up to crashed
```

--- a/package.json
+++ b/package.json
@@ -38,12 +38,12 @@
     "lodash.camelcase": "^4.3.0",
     "moment": "^2.22.2",
     "ts-node": "^8.1.0",
+    "tsconfig-paths": "^3.9.0",
     "typescript": "^3.1.6"
   },
   "devDependencies": {
     "jest": "^25.1.0",
     "ts-jest": "^25.1.0",
-    "tsconfig-paths": "^3.9.0",
     "tslint": "^5.11.0"
   }
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "hackbot",
-  "version": "2.3.0",
+  "version": "2.3.1",
   "description": "Discord bot for the Cascades Tech Club Discord server.",
   "repository": {
     "type": "git",

